community-labs/README.md

```diff
@@ -34,3 +34,4 @@ problem, refer directly to the authors.
 | UNLU Laboratorio de Redes                           | Kathara Labs - Course of "Network Fundamentals"     | A collection of Kathará network scenarios used in the Course of "Network Fundamentals" at National University of Luján, Argentina                                                                                                    | [Link](https://github.com/redesunlu/kathara-labs)                                         |  
 | Riccardo Gobbato                       | Kathara-P4 Forwarding-NSH-SFC     | Implementation of P4 program to perform packet forwarding according to the Service Function Chain (SFC) architecture, based on the Network Service Header (NSH).                                                                                                    | [Link](https://github.com/RicGobs/Kathara-P4-Forwarding-NSH-SFC)    
 | Mario Cuomo                       | Covert Timing Channel usign Kathara     | Creation of a Covert Timing Channel in python3 and kathará.                                                                                                    | [Link](https://github.com/mariocuomo/covert-timing-channel)   
+| Gabriele Alesci                       | KathaRange - a Kathara Cyber Lab     | A kathará lab featuring security open-source tools like Snort3, Wazuh Siem, Caldera MITRE, Atomic Red Team tools.                                                                                                    | [Link](https://github.com/g4br-i/KathaRange)   
```

Comment:
Hi @g4br-i, please fix the following two typos before merging.
Anyway the lab looks super interesting! Thanks for the PR! :smile: 

- "a Kathara Cyber Lab" -> A Kathará Cyber Lab
- "A kathará lab featuring [...]" -> "A Kathará lab featuring [...]"